jameda_interface_config.js

```diff
@@ -1,9 +1,11 @@
 /* eslint-disable no-unused-vars, no-var, max-len */
+const DEFAULT_IMG_HOST = 'https://security.patientus.de';
 
 var interfaceConfig = {
     // TO FIX: this needs to be handled from SASS variables. There are some
     // methods allowing to use variables both in css and js.
     DEFAULT_BACKGROUND: '#474747',
+    DEFAULT_LOGO_URL: `${DEFAULT_IMG_HOST}/assets/img/general/patientus_logo.png`,
```

Comment:
This will always render the default logo (patientus_logo). The ticket is I guess to make the Logo displayed for default (patientus) and also for the White-Labels. Therefore, I guess you have to check the subdomain and then render the logo which will be displayed in Jitsi. You might have something like ``${DEFAULT_IMG_HOST}/assets/img/general/${LOGO_NAME}``.

Reply:
Yes, you have right that will be render the default logo but when we provide a value over the external api the logo will be changed (please have a look here: https://github.com/jameda-telemedicine/jitsi-meet/pull/33/files#diff-41c3cb130f8647313139eeb1e4780e36R363). As we discussed we want to provide logo through the iframe then solution with subdomain here is not a proper way.